README.md

```diff
@@ -21,21 +21,19 @@ and build the library with default configuration values, provide `MQTT_DO_NOT_US
 
 ### Platform Prerequisites
```

Comment:
Why does the `Build Unit Tests.` have a period at the end? Also, isn't it better to have the reference examples before the build instructions for unit tests?